Tasks/Assignment 2/taglines.js

```diff
@@ -0,0 +1,33 @@
+var taglines = [
+  {
+    name: "Spiderman",
+    tagline: "With great power, comes great responsibility.",
+  },
+  {
+    name: "Hulk",
+    tagline: "Don't make me angry. You wouldn't like me when I'm angry.",
+  },
+  {
+    name: "Batman",
+    tagline: "Error! Batman is not a superhero. Fight me!",
+  },
+  {
+    name: "Wolverine",
+    tagline: "Bub.",
+  },
+  {
+    name: "Captain America",
+    tagline: "Language!",
+  },
+];
+
+function createSuperHero(hero) {
+  const heroObject = taglines.find((tagline) => tagline.name === hero);
+  if (heroObject) return () => `${heroObject.name} | ${heroObject.tagline}`;
+  else throw new Error("hero not found.");
+}
+
+// var mySuperHero = createSuperHero("Ironman");
```

Comment:
never left commented code inside the codebase

Reply:
resolved